phys/module_pbl_driver.F

```diff
@@ -2038,6 +2066,40 @@ SUBROUTINE pbl_driver(                                          &
               ,IMS=ims,IME=ime,JMS=jms,JME=jme,KMS=kms,KME=kme      &
               ,ITS=its,ITE=ite,JTS=jts,JTE=jte,KTS=kts,KTE=kte      )
        ENDIF
+     ELSEIF(gwd_opt .EQ. 3) THEN
+             CALL gwdo_gsl(                                         &
+               U3D=u_phy,V3D=v_phy,T3D=t_phy                        &
+              ,QV3D=qv_curr                                         &
+              ,P3D=p_phy,P3DI=p8w,PI3D=pi_phy,Z=z                   &
+              ,RUBLTEN=rublten,RVBLTEN=rvblten,RTHBLTEN=rthblten    &
+              ,DTAUX3D_ls=dtaux3d_ls,DTAUY3D_ls=dtauy3d_ls          &
+              ,DTAUX3D_bl=dtaux3d_bl,DTAUY3D_bl=dtauy3d_bl          &
+              ,DTAUX3D_ss=dtaux3d_ss,DTAUY3D_ss=dtauy3d_ss          &
+              ,DTAUX3D_fd=dtaux3d_fd,DTAUY3D_fd=dtauy3d_fd          &
+              ,DUSFCG_ls=dusfcg_ls,DVSFCG_ls=dvsfcg_ls              &
+              ,DUSFCG_bl=dusfcg_bl,DVSFCG_bl=dvsfcg_bl              &
+              ,DUSFCG_ss=dusfcg_ss,DVSFCG_ss=dvsfcg_ss              &
+              ,DUSFCG_fd=dusfcg_fd,DVSFCG_fd=dvsfcg_fd              &
+              ,xland=xland,br=br                                    &
+              ,VAR2D=var2dls,OC12D=oc12dls                              &
+              ,OA2D1=oa1ls,OA2D2=oa2ls,OA2D3=oa3ls,OA2D4=oa4ls              &
+              ,OL2D1=ol1ls,OL2D2=ol2ls,OL2D3=ol3ls,OL2D4=ol4ls              &
+              ,VAR2Dss=var2dss,OC12Dss=oc12dss                      &
+              ,OA2D1ss=oa1ss,OA2D2ss=oa2ss,OA2D3ss=oa3ss,OA2D4ss=oa4ss  &
+              ,OL2D1ss=ol1ss,OL2D2ss=ol2ss,OL2D3ss=ol3ss,OL2D4ss=ol4ss  &
```

Comment:
You've made the rest of this quite tidy, would you line these up also, please.

Reply:
Done.